src/audio/types.h

```diff
@@ -57,26 +56,49 @@ class ChannelCount {
     static ChannelCount fromLayout(ChannelLayout layout) {
         switch (layout) {
         case ChannelLayout::Mono:
-            return ChannelCount(1);
+            return mono();
         case ChannelLayout::DualMono:
-            return ChannelCount(2);
+            return stereo();
         case ChannelLayout::Stereo:
-            return ChannelCount(2);
+            return stereo();
         }
         DEBUG_ASSERT(!"unreachable code");
     }
 
+    static constexpr ChannelCount mono() {
+        return ChannelCount(static_cast<value_t>(1));
```

Comment:
Is the explicit static_cast needed?

Reply:
Yes, to use the constexpr C-tor. I am not aware of a unsigned char literal that works the same.